package.json

```diff
@@ -11,8 +11,8 @@
     "NOTICE"
   ],
   "engines": {
-    "node": "^12 || ^14 || ^15 || ^16 || ^18",
-    "npm": "^6 || ^7 || ^8"
+    "node": "^12 || ^14 || ^15 || ^16 || ^18 || ^19",
```

Comment:
Any reason to bump this up to 19? 18 is still the most recommended and LTS.

Reply:
19 is released though so not see the reason why we don't allow it.